python/qa-lint/action.yaml

```diff
@@ -1,26 +1,44 @@
 name: Static analysis
+inputs:
+  python-version:
+    required: false
+    description: Version of python
+    default: "3.12.2"
+  poetry-version:
+    required: false
+    description: Version of python
```

Comment:
It probably should be changed to `Version of poetry` here.

Reply:
Not sure. It depends on which version you're using.
```
      - name: Run Linter 
        uses: PiwikPRO/actions/python/qa-lint@master
        with:
            python-version: "3.13.*"
```
You can define it dynamically in the action definition.

Reply:
ooh, you're right